content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -521,7 +521,214 @@ To verify your device status, click on your FoundriesFactory, go to **Devices**
+
+![FoundriesFactory device overview](assets/web_board_manager_factory_device-overview.png "FoundriesFactory device overview")
+
+If this is not the case, you can update your device using FoundriesFactory **Waves** functionality. Check [this tutorial](https://docs.arduino.cc/tutorials/portenta-x8/waves-fleet-managment) to read the complete instructions. More information about Waves can be found in the official Foundries documentation at [this link](https://docs.foundries.io/latest/reference-manual/factory/fioctl/fioctl_waves.html?highlight=wave).
+
+### Update For OS Release V.399
+
+If your Portenta X8 is flashed with the OS release V.399, open a new Command Line window and type the following commands on your PC:
+
+```arduino
+wget https://downloads.arduino.cc/portentax8image/update-latest.tar.gz
+```
+
+```arduino
+wget https://downloads.arduino.cc/portentax8image/aklite-offline-399.tar.gz
+```
+
+Previous commands will allow you to get the latest OS image within the `update-latest.tar.gz` and `aklite-offline-399.tar.gz` packages. If `wget` command is not recognized or causing trouble, you may use a browser and use the direct link without the `wget` command and download the files. The files should then be located inside the `adb` tool directory. Further detail regarding `ADB` is explained in [Working with Linux section](#working-with-linux) of this user manual.
+
+```arduino
+adb push update-latest.tar.gz /home/fio
+```
+
+```arduino
+adb push aklite-offline-399.tar.gz /home/fio
+```
+
+These commands will make your V.399 compatible with [aklite-offline](https://docs.foundries.io/latest/user-guide/offline-update/offline-update.html) tool and will allow you to update your Portenta X8 to the latest image version Arduino released at that point in time. Arduino provides this tool for free for any Portenta X8 user to enable offline secure updates to all devices, even if those devices are not connected to any FoundriesFactory.
+
+After the updates have been correctly downloaded to your PC, you can open a Command Line window and connect to your Portenta X8 through `ADB Shell`, as explained in [this section](#working-with-linux) of this user manual.
+
+Once your Portenta X8 is correctly connected to your PC, launch the following commands in sequence to update your device to the latest released OS image version:
+
+```arduino
+adb shell
+```
+
+```arduino
+cd /home/fio
+```
+
+```arduino
+tar -xvf update-latest.tar.gz -C .
+```
+
+```arduino
+tar -xvf aklite-offline-399.tar.gz -C .
+```
+
+```arduino
+export LD_LIBRARY_PATH=usr/lib/
+```
+
+```arduino
+sudo aklite-offline install --src-dir /var/rootdirs/home/fio/offline-updates/
+```
+
+After the update process is finalized, you need to restart your Portenta X8 by pressing its button for around 10 seconds. Once restarted, your Portenta X8 will immediately start running the latest OS release.
+
+### Update Using `uuu` Command
```

Comment:
Can we split this section into two subsections "Flashing with carrier" and "Flashing without carrier" as you have already done [here](https://github.com/arduino/docs-content/pull/948/files)?

Reply:
Sure, we can do that. I'll make the change, and since we have the content already applied, I'll make adjustments to fit it.